Makefile.core.mk

```diff
@@ -398,6 +398,10 @@ gen-api-docs: ## Generate API documentation.
 		--output-path=$(OUTPUT_DOCS_PATH) \
 		--output-mode=group
 	@echo "API reference documentation generated at $(OUTPUT_DOCS_PATH)"
+	## The package used to convert CRDs to markdown files generates tabs in the output markdown files and br.
+	## This command replaces tabs with two spaces and removes br tags.
+	@find $(OUTPUT_DOCS_PATH) -type f -name "*.md" -exec sed -i 's/\t/  /g' {} \;
+	@find $(OUTPUT_DOCS_PATH) -type f -name "*.md" -exec sed -i 's/<br>/\n/g;s/<br \/>/\n/g' {} \;
```

Comment:
Is this the right place to fix the formatting, given that you're only fixing the reference docs and not, for example, the `kubectl explain` output?

Reply:
See description field when running `kubectl explain istio.spec.values.meshConfig.meshMTLS`. The example has tabs instead of spaces.

Reply:
I'll check to do the change in the transform API to see if we can fix this in all the places then